modules/ROOT/partials/configuration/sandbox_iframes.adoc

```diff
@@ -1,5 +1,5 @@
-[[sandbox-iframes-option]]
-== `sandbox_iframes` option
+[[sandbox-iframes]]
+== `sandbox_iframes`
 
 This option controls whether the editor will add a `sandbox=""` attribute to all `<iframe>` elements. This will restrict the iframe’s embedded resource from performing potentially malicious actions including scripting, file downloads, browser popups, passing the same-origin policy, among others. Reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox[MDN].
```

Comment:
```suggestion
This option allows you to control whether `<iframe>` elements are [sandboxed](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/iframe#sandbox) when inserted into the editor. When enabled, all `<iframe>` elements will be given the `sandbox=""` attribute, applying all restrictions.
```
We don't want to describe how to perform an attack. Sorry I missed this during the initial review.